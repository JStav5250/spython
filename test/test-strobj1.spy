str x = "plt2024"
print(x)
print(x[0])
print(x[0][0])
