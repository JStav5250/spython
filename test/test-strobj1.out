plt2024
p
p
